Source/Fuse.LocalNotifications/Android/LocalNotificationReceiver.java

```diff
@@ -12,4 +13,19 @@ public void onReceive(Context context, Intent intent)
     {
         com.foreign.Fuse.LocalNotifications.AndroidImpl.OnNotificationRecieved(context, intent);
     }
+
+	public static String MakePayloadString(String title, String body, String payload)
+	{
+		JSONObject jsonObj = new JSONObject();
+		try
+		{
+			jsonObj.put("title", (title == null ? "" : title));
+			jsonObj.put("body", (body == null ? "" : body));
+			jsonObj.put("payload", (payload == null ? "" : payload));
+		}
+		catch (Exception e)
+		{
```

Comment:
Empty catch-block... really? If that's a sane thing to do, maybe add a comment to explain why?

Reply:
If an exception triggers, the body won't run fully, leaving us with a half-filled in jsonObj, no? That sounds bad to me...